### ISSUE TITLE
Discussion for To-Do should have markdown support 

### ISSUE THREAD

**Issue by cesalo** [Leantime/leantime#246]
**Is your feature request related to a problem? Please describe.**
Discussion tab in To-do tasks should have markdown or some options to add some formatting like bold letters, bullet points, etc.
**Describe the solution you'd like**
To-do tasks should have markdown or some options to add some formatting like bold, bullet points, etc. Like when creating issues in Github or any other similar app.

**Comment by marcelfolaron** [comment]
I agree! Anyone have a good suggestion for a solid open source markdown editor? We've been debating switching to ckeditor since that comes with markdown support but it's still quite heavy for what we need.

**Comment by jboilesen** [comment]
@marcelfolaron , in that case, I recommend SimpleMDE.

https://simplemde.com/

**Comment by marcelfolaron** [comment]
Integrated tinymce for formatting

**Comment by aurora** [comment]
Mmm ... it would be cool if i could edit in source-mode - it seems to be impossible to format using a fixed-width font eg. for pasting code snippets.
